### DISCUSSION TITLE
Dialogflow Integration

### DISCUSSION THREAD

**Discussion by Piyush0125** [LiveHelperChat/livehelperchat#1924] [Q&A] [answered]
How to integrate dialogflow in live helper chat 
kindly please clear me the steps regarding these

**Comment by remdex** [comment, marked as answer]
Same flow as here
https://doc.livehelperchat.com/docs/bot/rasa-integration

Read article
https://doc.livehelperchat.com/docs/bot/rest-api#events-you-can-listen

**Reply by Piyush0125** [reply]
Thank you remdex
